kafka-native/src/main/java/org/ballerinalang/messaging/kafka/utils/KafkaConstants.java

```diff
@@ -34,7 +34,7 @@ private KafkaConstants() {
     public static final int DURATION_UNDEFINED_VALUE = -1;
 
     // Kafka log messages
-    public static final String SERVICE_STARTED = "[ballerinax/kafka] started kafka listener ";
+    public static final String SERVICE_STARTED = "[ballerinax/kafka] started kafka listener for topic/s ";
```

Comment:
How about "Kafka listener started for topics: " WDYT? 
Also WDYT @ThisaruGuruge

Reply:
sounds much better

Reply:
That would be more consistent with rabbitmq https://ballerina.io/learn/by-example/rabbitmq-consumer.html

Reply:
+1

Reply:
Updated now